umh.docs.umh.app/content/en/docs/production-guide/upgrading/upgrading.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Management Console Upgrades
+content_type: task
+description: |
+  This page describes how to perform the upgrades that are available for the Management Console.
+weight: 2000
+---
+
+<!-- overview -->
+
+The Management Console simplifies the upgrade process for both the Management Companion and the UMH stack (not yet available).
+This page provides detailed instructions on executing these upgrades.
```

Comment:
Here I'd just mention that the user can use the Console to upgrade their instance, and that this page provides the instructions